lib/redis.js

```diff
@@ -46,10 +63,8 @@ function createClient(config, app) {
   }
 
   app.beforeStart(async () => {
-    const result = await client.info();
```

Comment:
After this change, we do not have any check before the start? I think it's a breaking change and properly we shouldn't do that.

Reply:
`result[0]` is just  a '#' character which has no sense in the log file